src/main/java/service/ReportCreator.java

```diff
@@ -0,0 +1,22 @@
+package service;
+
+import db.Storage;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+public class ReportCreator {
+    private static final String HEADER = "fruit,quantity";
+
+    public List<String> create() {
```

Comment:
It is better to return the Report as String
```suggestion
    public String prepareReport() {
```